traefik/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
-version: 35.3.0
+version: 35.4.0
```

```suggestion
version: 35.3.0
```
Please do not touch the version. Release are done following a specific process.